rust-app/src/menu.rs

```diff
@@ -85,16 +106,59 @@ impl Menu for IdleMenuWithSettings {
         }
     }
     #[inline(never)]
-    fn label(&self) -> &str {
+    fn label<'a>(&self) -> (MenuLabelTop<'a>, MenuLabelBottom<'a>) {
```

Comment:
Clippy probably will want
```suggestion
    fn label(&self) -> (MenuLabelTop<'_>, MenuLabelBottom<'_>) {
```

Reply:
Nope, clippy is fine